src/Esfa.Vacancy.Application/Commands/CreateApprenticeship/Validators/Extensions.cs

```diff
@@ -21,6 +23,15 @@ public static class Extensions
                 .WithMessage(string.Format(ErrorMessages.CreateApprenticeship.Whitelist, propertyName));
         }
 
+        public static IRuleBuilderOptions<CreateApprenticeshipRequest, string> MatchesAllowedFreeTextCharacters(
```

Comment:
Nice. 

I'll leave the code as is for the overload I wrote so that I don't need to change all other usages of `ErrorMessages.CreateApprenticeship.Whitelist` for now. We can combine the approaches in a subsequent story to make some changes to the validation in general. E.g. move existing validators into the "new" way of doing things (using the partial class).